### DISCUSSION TITLE
求加入vim编辑功能

### DISCUSSION THREAD

**Discussion by ThomasLi10** [tw93/MiaoYan#244] [Q&A]
求加入vim编辑功能

**Comment by jackytang** [comment]
话说我也好希望有这个功能, 其实能以插件方式提供也不错

**Comment by tw93** [comment]
很抱歉，我有调研过在Mac原生应用的原生组件NSTextView里面支持Vim模式相当于重写了一个Vim引擎，还是很复杂的，暂时我这边自己用不太上这个功能，同时很容易增加代码复杂维护，假如有兴趣，欢迎去试试，可以给我提交代码
